TestForSignUp/InputTextGenerator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace TestForSignUp
+{
+    internal class InputGenerator
+    {
+        internal string emailString;
+        internal string dateString;
+        internal string commonString;
+
+        internal InputGenerator()
+        {
+            emailString = Guid.NewGuid().ToString() + "@gmail.com";
```

Comment:
I understand that this is unlikely, but what happens if we create a user who is already in db?

Reply:
> I understand that this is unlikely, but what happens if we create a user who is already in db?

Techically that's why email is generated with new guid each time, to avoid creating same user